scripts/e2e-ephemeral-test.sh

```
@@ -104,21 +104,21 @@ log_info "Database pod: $DB_POD"
 # Step 3: Connect to DB and verify all tables are empty
 log_info "Verifying database tables are empty..."
 
-TABLES=("resource" "reporter_resources" "reporter_representations" "common_representations" "outbox_events")
+TABLES=("resource" "reporter_resources" "reporter_representations" "common_representations")
 
 for table in "${TABLES[@]}"; do
     log_info "Checking table: $table"
     COUNT=$(oc exec -n $NAMESPACE $DB_POD -- bash -c "psql -U postgres -d kessel-inventory -t -c \"SELECT COUNT(*) FROM $table;\"" 2>&1)
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)